common/src/java/com/zimbra/common/soap/Element.java

```diff
@@ -257,7 +257,13 @@ public String getQualifiedName() {
 
     public QName getQName() {
         String uri = getNamespaceURI(mPrefix);
-        return uri == null ? QName.get(mName) : QName.get(getQualifiedName(), uri);
+        QName returnValue = null;
+        try {
+            returnValue = (uri == null) ? QName.get(mName) : QName.get(getQualifiedName(), uri);
+        } catch (Exception e) {
```

Comment:
Can we catch specific exception that is thrown by dom4j?

Reply:
Done